.github/workflows/build-simtools-production-image.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        simtel_version: ['240205', '240927', '250304']
+        simtel_version: ['240927', '250304']
```

Comment:
Please do not remove the `240205` just yet. That is the tarball which is equivalent to the Prod6 package on cvmfs. I suggest we keep it until we have a thorough validation of the simtools 6.0.0 production.

Reply:
Did we ever build it? Note above change removes it from the list of tar packages to be downloaded, but the tar packages was actually not used.

So I've added building it.

Reply:
No, we've never used it in simtools. But if we ever need to debug when we do statistical comparisons, I can imagine it might be useful (and it doesn't cost anything to keep for now).